jupyter/datascience/anaconda-python-3.8/environment.yml

```diff
@@ -0,0 +1,363 @@
+  - pyflakes=3.0.1=pyhd8ed1ab_0
+  - pygithub=1.58.0=pyh1a96a4e_0
+  - pygments=2.15.1=pyhd8ed1ab_0
+  - pyjwt=2.7.0=pyhd8ed1ab_0
+  - pylint=2.17.4=pyhd8ed1ab_0
+  - pynacl=1.5.0=py38h0a891b7_1
+  - pyopenssl=20.0.1=pyhd8ed1ab_0
+  - pyparsing=3.0.9=pyhd8ed1ab_0
+  - pyqt=5.15.7=py38h6a678d5_1
+  - pyqt5-sip=12.11.0=py38h6a678d5_1
+  - pyrsistent=0.18.0=py38heee7806_0
+  - pysocks=1.7.1=pyha2e5f31_6
+  - python=3.8.16=h7a1cb2a_3
+  - python-dateutil=2.8.2=pyhd8ed1ab_0
+  - python-fastjsonschema=2.17.1=pyhd8ed1ab_0
+  - python-flatbuffers=23.5.26=pyhd8ed1ab_0
+  - python-json-logger=2.0.7=pyhd8ed1ab_0
+  - python-kubernetes=18.20.0=pyhd8ed1ab_0
+  - python-lsp-jsonrpc=1.0.0=pyhd8ed1ab_0
+  - python-lsp-server=1.7.3=pyhd8ed1ab_0
+  - python-lsp-server-base=1.7.3=pyhd8ed1ab_0
+  - python_abi=3.8=2_cp38
+  - pytoolconfig=1.2.5=pyhd8ed1ab_0
+  - pytorch=1.13.1=cpu_py38h9dc8d95_0
+  - pytz=2023.3=pyhd8ed1ab_0
+  - pyu2f=0.1.5=pyhd8ed1ab_0
+  - pyyaml=5.4.1=py38h497a2fe_1
+  - pyzmq=25.0.2=py38h6a678d5_0
+  - qt-main=5.15.2=h8373d8f_8
+  - qt-webengine=5.15.9=hbbf29b9_6
+  - qtwebkit=5.212=h3fafdc1_5
+  - re2=2022.04.01=h27087fc_0
+  - readline=8.2=h5eee18b_0
+  - requests=2.31.0=pyhd8ed1ab_0
+  - requests-oauthlib=1.3.1=pyhd8ed1ab_0
+  - requests-toolbelt=0.10.1=pyhd8ed1ab_0
+  - rfc3339-validator=0.1.4=pyhd8ed1ab_0
+  - rfc3986-validator=0.1.1=pyh9f0ad1d_0
+  - rich=13.4.1=pyhd8ed1ab_0
+  - rope=1.8.0=pyhd8ed1ab_0
+  - rsa=4.9=pyhd8ed1ab_0
+  - scikit-learn=1.0.2=py38h1561384_0
+  - scipy=1.8.1=py38h1ee437e_0
+  - send2trash=1.8.2=pyh41d4057_0
+  - setuptools=66.0.0=py38h06a4308_0
+  - shellingham=1.5.1=pyhd8ed1ab_0
+  - simpervisor=1.0.0=pyhd8ed1ab_0
+  - simplejson=3.17.6=py38h0a891b7_1
+  - sip=6.6.2=py38h6a678d5_0
+  - six=1.16.0=pyh6c4a22f_0
+  - smmap=3.0.5=pyh44b312d_0
+  - snappy=1.1.9=h295c915_0
+  - sniffio=1.3.0=pyhd8ed1ab_0
+  - snowballstemmer=2.2.0=pyhd8ed1ab_0
+  - soupsieve=2.3.2.post1=pyhd8ed1ab_0
+  - sqlite=3.41.2=h5eee18b_0
+  - stack_data=0.6.2=pyhd8ed1ab_0
+  - strip-hints=0.1.8=py_0
+  - tabulate=0.9.0=pyhd8ed1ab_1
+  - tenacity=8.2.2=pyhd8ed1ab_0
+  - tensorboard=2.12.1=py38h06a4308_0
+  - tensorboard-data-server=0.7.0=py38h52d8a92_0
+  - tensorboard-plugin-wit=1.8.1=pyhd8ed1ab_0
+  - tensorflow=2.12.0=mkl_py38h496960e_0
+  - tensorflow-base=2.12.0=mkl_py38he5f8e37_0
+  - tensorflow-estimator=2.12.0=py38h06a4308_0
+  - termcolor=2.3.0=pyhd8ed1ab_0
+  - terminado=0.17.1=pyh41d4057_0
+  - textwrap3=0.9.2=py_0
+  - threadpoolctl=3.1.0=pyh8a188c0_0
+  - tinycss2=1.2.1=pyhd8ed1ab_0
+  - tk=8.6.12=h1ccaba5_0
+  - toml=0.10.2=pyhd8ed1ab_0
+  - tomli=2.0.1=pyhd8ed1ab_0
+  - tomlkit=0.11.8=pyha770c72_0
+  - tornado=6.2=py38h5eee18b_0
+  - tqdm=4.65.0=pyhd8ed1ab_1
+  - traitlets=5.9.0=pyhd8ed1ab_0
+  - typer=0.9.0=pyhd8ed1ab_0
+  - typing-extensions=4.6.3=hd8ed1ab_0
+  - typing_extensions=4.6.3=pyha770c72_0
+  - typing_utils=0.1.0=pyhd8ed1ab_0
+  - ujson=5.3.0=py38hfa26641_0
+  - uritemplate=3.0.1=py_0
+  - urllib3=2.0.2=pyhd8ed1ab_0
```

Comment:
lets pin urllib3 under  <1.26.16, as they have removed some function in 2.0 version , which being used in elyra.